src/Informer/ConsoleMigrationInformer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration\Informer;
+
+use Yiisoft\Yii\Db\Migration\Helper\ConsoleHelper;
+
+final class ConsoleMigrationInformer implements MigrationInformerInterface
+{
+    private ConsoleHelper $helper;
+
+    public function __construct(ConsoleHelper $helper)
+    {
+        $this->helper = $helper;
+    }
+
+    public function info(int $type, string $message): void
+    {
+        switch ($type) {
```

Comment:
How about making four separate methods instead of using constants? This way we'll get rid of `MigrationInformerType` and require supporting all the message types in `MigrationInformerInterface`.

Reply:
Good idea! I did it.